_posts/2021-06-30-join-demodam-hackathon.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Join the Demodam hackathon"
+date: 2021-06-30
+author: Alba Roza
+type: blogpost
+excerpt: Information on how to join the following Demodam hackathon
+categories:
+  - News
+---
+
+# Join the Demodam hackathon
+
+Demodam is a new concept that brings together a community of developers, companies, municipalities and users. The idea of Demodam, a demo of a municipal website, enables reusable open source products and services available for municipalities in a single market place.
+
+## The hackathon
+
+During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here are the details you need to know.
```

Comment:
```suggestion
During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here's what you need to know.
```

Reply:
Suggest replacing "would you like to contribute" with a more concrete invitation - have you previously worked on a Common Ground component? Are you a dev (or whoever you're hoping to attract?) We'd love to have you!